### PR TITLE
Registration and TumbleBitTests had missing references.

### PR DESCRIPTION
**Pull request by ferdeen**
Fixed to build the solution.

### CODE REVIEW

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs [hunk 1]
```diff
@@ -147,7 +147,7 @@ public void TestWithTor()
                 CoreNode node1 = builder.CreateStratisPowNode(true, fullNodeBuilder =>
                 {
                     fullNodeBuilder
-                        .UseConsensus()
+                        .UsePosConsensus()
```
Comment (zeptin) [inline Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: Should be UsePowConsensus here; TB runs on the BTC network at present

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs [hunk 2]
```diff
@@ -406,7 +406,7 @@ public void TestWithoutTor()
...
```
Comment (zeptin) [inline Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: Pow

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs [hunk 3]
```diff
@@ -661,7 +661,7 @@ public void TestDualClientWithoutTor()
                 CoreNode node1 = builder.CreateStratisPowNode(false, fullNodeBuilder =>
                 {
                     fullNodeBuilder
-                        .UseConsensus()
+                        .UsePosConsensus()
```
Comment (zeptin) [inline Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: Pow

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs [hunk 4]
```diff
@@ -679,7 +679,7 @@ public void TestDualClientWithoutTor()
                 CoreNode node2 = builder.CreateStratisPowNode(false, fullNodeBuilder =>
                 {
                     fullNodeBuilder
-                        .UseConsensus()
+                        .UsePosConsensus()
```
Comment (zeptin) [inline Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: Pow

File: Breeze.Registration.Tests/RegistrationTests.cs [hunk 5]
```diff
@@ -167,7 +158,6 @@ public void RegistrationTest()
                 };
 
                 var tx = wth1.BuildTransaction(context);                
-                node1.AddToStratisPosMempool(tx);
```
[the remark this answers is not in the document]
Comment (ferdeen) [reply Breeze.Registration.Tests/RegistrationTests.cs]: ok

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs (hunk 1 above)
Comment (ferdeen) [reply Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: done

Comment (ferdeen) [reply Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: done

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs (hunk 2 above)
Comment (ferdeen) [reply Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: done

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs (hunk 3 above)
Comment (ferdeen) [reply Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: done

File: Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs (hunk 4 above)
Comment (ferdeen) [reply Breeze.TumbleBit.Client.Tests/TumbleBitTests.cs]: done

**Pull request merged**